.github/workflows/python-versions-runner.yml

```diff
@@ -0,0 +1,31 @@
+name: Python versions runner
+on:
+  workflow_dispatch:
+    inputs:
+      versions:
+        description: 'Versions to build'
+        required: true
+        default: '","'
+      publish-releases:
```

Comment:
Looks like we don't use this input at all

Reply:
Indeed